CMakeLists.txt

```diff
@@ -10844,11 +10844,14 @@ target_link_libraries(grpc_cpp_plugin
 
 
 if(gRPC_INSTALL)
-  install(TARGETS grpc_cpp_plugin EXPORT gRPCTargets
+  install(TARGETS grpc_cpp_plugin
     RUNTIME DESTINATION ${gRPC_INSTALL_BINDIR}
     LIBRARY DESTINATION ${gRPC_INSTALL_LIBDIR}
     ARCHIVE DESTINATION ${gRPC_INSTALL_LIBDIR}
   )
+  if(NOT CMAKE_CROSSCOMPILING)
+    install(TARGETS grpc_cpp_plugin EXPORT gRPCTargets)
```

Comment:
what's the reasoning behind  not exporting grpc_cpp_plugin as gRPCTargets (as opposed to simply skipping the `install(TARGETS grpc_cpp_plugin ...)` entirely when crosscompiling)?

Reply:
We can do that too. You think it would be better for consistency between native and cross-compilation builds?